config/environments/test.rb

```diff
@@ -63,5 +63,5 @@
   # Disable logging
   config.logger = ActiveSupport::TaggedLogging.new(Insights::Api::Common::LoggerWithAudit.new(File::NULL))
 
-  config.factory_bot.definition_file_paths = ["test/factories"]
+  config.factory_bot.definition_file_paths = ["test/factories", "spec/factories"]
```

Comment:
The `spec/factories` here will eventually (in follow-up PRs) replace the `test/factories`.

Am I understanding it right, @xmicha82?

Reply:
Yes!

Reply:
well, `test/factories` are for APIv1 tests, aren't they? so i don't think they will be replaced